scripts/updateProdEnv.ts

```diff
@@ -0,0 +1,32 @@
+import fs from "fs";
+
+const HF_TOKEN = process.env.HF_TOKEN;
+const SECRET_CONFIG = fs.existsSync(".env.SECRET_CONFIG")
+	? fs.readFileSync(".env.SECRET_CONFIG", "utf8")
+	: process.env.SECRET_CONFIG;
+
+if (SECRET_CONFIG === "") {
+	throw new Error(
+		"SECRET_CONFIG is not defined. Please provide it either in a file or as an environment variable."
+	);
+}
+
+// Read the content of the file .env.template
+const PUBLIC_CONFIG = fs.readFileSync(".env.template", "utf8");
+
+// Prepend the content of the env variable SECRET_CONFIG
+const full_config = `${PUBLIC_CONFIG}\n${SECRET_CONFIG}`;
+
+// Make an HTTP POST request to add the space secre
+fetch(`https://huggingface.co/api/spaces/huggingchat/chat-ui/secrets`, {
```

Comment:
I don't think `@huggingface/hub` supports the spaces secrets API yet so I made a manual API call here. Tested it on my own space and it works 

@coyotte508 for visibility, not sure if it's planned or not haha